src/condensed.rs

```diff
@@ -47,12 +51,13 @@ impl<'a, T> CondensedMatrix<'a, T> {
             assert!(observations <= 1);
             CondensedMatrix { data: data, observations: 0 }
         } else {
+            assert!(observations >= 2);
             assert_eq!((observations * (observations - 1)) / 2, data.len());
             CondensedMatrix { data: data, observations: observations }
         }
     }
 
-    /// Return the number of observations reflexed by this matrix.
+    /// Return the number of observations reflected by this matrix.
```

Comment:
Minor point: When I first read this, I thought "reflected" was a technical word. But, now I realize it is not.

Reply:
Interesting. I think I just didn't want to say "the number of observations in the matrix" since that doesn't seem quite right. I instead just switched to more mundane language:

> Return the number of observations that make up this matrix.